assign2/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="col-1">
+        <div class="head">
+          <h1>Why Loconav's Tracking System?</h1>
+        </div>
+        <div class="d1">
+          <img id="i1" src="topl.png" />
+          <h2>INTELLIGENT VEHICLE TRACKING</h2>
+          <p>
+            Whether you are a single vechicle or a
+            <a href="#">fleet owner</a> . Loconav is determined to protect and
+            monitor your asset from theft or harm.
+          </p>
+        </div>
+        <div class="d2">
+          <img id="i3" src="topr.png" />
+          <h2>VEHICLE LOCK</h2>
+          <p>
+            Turn on vehicle lock form loconav <a href="#">mobile app</a> and be
+            assured that your vehicle wont start without your permission.
+          </p>
+        </div>
+        <div class="d3">
+          <img id="i2" src="btml.png" />
+          <h2>LIVE ALERTS</h2>
+          <p>
+            Get live over speeding alerts on entry and exit points, monitor
+            idling in real time etc.Vehicle services and maintainance alerts.
+          </p>
+        </div>
+        <div class="d4">
+          <img id="i4" src="btmr.png" />
+          <h2>VEHICLE HEALTH</h2>
+          <p>
+            Get diagnostic,mileage and health report of your vehicle so that you
+            can worry less about vechicle's well being and enjoy your time on
+            road
+          </p>
+        </div>
+      </div>
+      <div class="col-2">
+        <div class="form">
+          <form>
+            <h2>Get an offer on your mobile</h2>
+            <input type="text" placeholder="Phone Number" /><br />
+            <input type="text" placeholder="How Many Vehicles?" />
+            <div class="sel">Select Your Vehicle</div>
+            <div class="options">
+              <button>Car</button>
+              <button>Truck</button>
+              <button>Others</button>
+            </div>
+            <button class="submit-btn">GET STARTED</button>
+            <p>Call Us: +91 8448881414</p>
+            <p></p>
```

Comment:
Why this blank tag? Just for spacing? Can't we use something else here?

Reply:
removed